lib/fuser_basic.h

```diff
@@ -31,26 +31,35 @@ template <typename IO, typename Gate>
 struct BasicGateFuser final {
   using GateFused = qsim::GateFused<Gate>;
 
+  /**
+   * User-specified parameters for gate fusion.
+   * BasicGateFuser does not use any parameters.
+   */
+  struct Parameter {};
```

Comment:
I assume fusion with more than two qubits will be handled by a new Fuser type?

Reply:
Yes, exactly.